src/latin_language_model.js

```diff
@@ -194,7 +194,10 @@ export default class LatinLanguageModel extends LanguageModel {
       pronounClassRequired: false
     }
     if (inflection.hasOwnProperty(Feature.types.part)) {
-      if (inflection[Feature.types.part].value === Constants.POFS_PRONOUN) {
+      if (inflection[Feature.types.part].value === Constants.POFS_VERB) {
+        grammar.fullFormBased = true
+        grammar.suffixBased = true
+      } if (inflection[Feature.types.part].value === Constants.POFS_PRONOUN) {
```

Comment:
Although it would probably not matter in real life situations, maybe it's better to put `else if` here for more clarity